addons/portal_rating/views/rating_templates.xml

```diff
@@ -51,7 +51,7 @@
                      the message will be posted with the identity of the partner_id of the object
     -->
     <template id="rating_stars_static_popup_composer" name="Rating: rating composer in popup">
-        <t t-set="display_composer" t-value="not disable_composer and not request.session.is_website_user"/>
+        <t t-set="display_composer" t-value="not disable_composer"/>
```

Comment:
@std-odoo I forgot why I needed to remove it...Do you remember ?

Reply:
I think it was because is_website_user was not in the dict session, I think you should replace it with this
![screenshot_spect](https://user-images.githubusercontent.com/54316226/150142187-924ec124-9774-466b-a339-6d59a4eb3ecf.png)

Reply:
@Julien00859 up